tutor/commands/jobs.py

```diff
@@ -308,12 +309,72 @@ def sqlshell(args: list[str]) -> t.Iterable[tuple[str, str]]:
     Extra arguments will be passed to the `mysql` command verbatim. For instance, to
     show tables from the "openedx" database, run `do sqlshell openedx -e 'show tables'`.
     """
-    command = "mysql --user={{ MYSQL_ROOT_USERNAME }} --password={{ MYSQL_ROOT_PASSWORD }} --host={{ MYSQL_HOST }} --port={{ MYSQL_PORT }} --default-character-set=utf8mb3"
+    command = "mysql --user={{ MYSQL_ROOT_USERNAME }} --password={{ MYSQL_ROOT_PASSWORD }} --host={{ MYSQL_HOST }} --port={{ MYSQL_PORT }} --default-character-set=utf8mb4"
     if args:
         command += " " + shlex.join(args)  # pylint: disable=protected-access
     yield ("lms", command)
 
 
+   
+@click.command(
+    short_help="Upgrade mysql to a specific charset and collation",
+    help=(
+        "Upgrade mysql to a specific charset and collation. You can either upgrade all tables, specify only certain tables to upgrade or specify certain tables to exclude from the upgrade process"
+    ),
+    context_settings={"ignore_unknown_options": True},
+)
+@click.option("--all-tables", is_flag=True, help="change all tables in the openedx database")
+@click.option("-s", "--status", type=click.Choice(['include', 'exclude'], case_sensitive=False), help="Whether to include or exclude certain tables/apps")
```

Comment:
This combination of options is not very intuitive. It means that the end users needs to specify at least one option -- and that is not a great pattern, as options are considered... optional :)

Instead, I suggest that:

1. all tables are process by default.
2. If `--include` is specified, then only those tables are processed.
3. If `--exclude` is specified, then those tables are not processed.

Note that with this approach, `--include` and `--exclude` are mutually compatible (`--exclude` would override `--include`).

Reply:
> Maybe we can prompt the user if they wish to proceed?

Yes, this sounds perfect. Please make sure to include a `-I/--non-interactive` option for headless scripts.